lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -313,8 +314,8 @@ void FIRRTLTypesLowering::visitDecl(MemOp op) {
         // to all clocks. This is handled by creating a dummy wire,
         // setting the dummy wire as the lowering target, and then
         // connecting every new port subfield to that.
-        if (elt.name == "clk" | elt.name == "en" | elt.name == "addr" |
-            elt.name == "wmode") {
+        if ((elt.name == "clk") | (elt.name == "en") | (elt.name == "addr") |
```

Comment:
Driveby: Please use || for consistency, and for the usual precedence level that doesn't need parens :-)